fixtures/docstring-proc-macro/tests/test_generated_bindings.rs

```diff
@@ -6,6 +6,10 @@ uniffi::build_foreign_language_testcases!(
 
 #[cfg(test)]
 mod tests {
+    use uniffi_bindgen::bindings::kotlin::gen_kotlin::KotlinBindingGenerator;
```

Comment:
Do you mind updating these to import from `uniffi_bindgen::bindings`?

Reply:
✅